StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1519BracesMustNotBeOmittedFromMultiLineChildStatement.cs

```diff
@@ -108,9 +109,15 @@ private static void HandleForEachStatement(SyntaxNodeAnalysisContext context)
             CheckChildStatement(context, forEachStatement.Statement);
         }
 
-        private static void HandleUsingStatement(SyntaxNodeAnalysisContext context)
+        private static void HandleUsingStatement(SyntaxNodeAnalysisContext context, StyleCopSettings settings)
         {
             var usingStatement = (UsingStatementSyntax)context.Node;
+
+            if (settings.LayoutRules.AllowConsecutiveUsings && (usingStatement.Statement is UsingStatementSyntax))
```

Comment:
That would not be allowed, as the code explicitly checks if the child statement is a using statement.

StyleCop classic also does not allow that construction.